stix2validator/validator.py

```diff
@@ -110,8 +111,38 @@ def as_json(self):
         return json.dumps(self.as_dict())
 
 
-class ValidationResults(BaseResults):
-    """Results of JSON schema validation.
+class FileValidationResults(BaseResults):
+    """
+    Represents validation results for a file.  This entails potentially
+    several STIX object results, since a file may contain a list of STIX
+    objects.
+    """
+    def __init__(self, is_valid=False, filepath=None, object_results=None, fatal=None):
+        """
+        Initialize this instance.
+        :param is_valid: Whether the overall result is valid
```

Comment:
At some point I'd rather switch to using [Google-style docstrings](http://www.sphinx-doc.org/en/stable/ext/napoleon.html#google-vs-numpy) (across everything). It doesn't need to be now, but I'd rather new docstrings use this format.

Reply:
Pycharm always autocompletes docstrings in rst style from what I've seen, and it just seems like a lot of trouble to rewrite, for not enough benefit.

Reply:
Interesting, I didn't know that about PyCharm. I personally find the Google-style ones much more readable. Not worth changing at this point; I'm happy there are docstrings at all 😉 .